test/AutoRest.TestServer.Tests/TypeAsserts.cs

```diff
@@ -69,6 +69,53 @@ public static void TypeOnlyDeclaresThesePublicMethods(Type type, params string[]
             Assert.Fail($"Type \"{type}\" doesn't have public methods {missingMethods} and has public methods {extraMethods} that aren't expected.");
         }
 
+        public static void HasPublicInstanceMethod(Type type, string methodName, Parameter[] parameters)
```

Comment:
@annelo-msft , can you please check the messages in this assertion method?

Reply:
These look good to me!